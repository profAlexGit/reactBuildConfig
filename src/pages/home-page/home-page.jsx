import React from 'react';
import Header from '../../components/header/Header.jsx';

export const HomePage = () => {
  return (
    <main>
      <Header />
      <h1>Home Page</h1>
    </main>
  );
};

export default HomePage;
