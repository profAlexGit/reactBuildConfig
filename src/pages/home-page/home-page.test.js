import React from 'react';
import { configure } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import Adapter from 'enzyme-adapter-react-16';

import { HomePage } from './home-page.jsx';
configure({ adapter: new Adapter() });

describe('<HomePage />', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<HomePage />);
    expect(tree).toMatchSnapshot();
  });
});
