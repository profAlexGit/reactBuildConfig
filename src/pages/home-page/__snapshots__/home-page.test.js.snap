// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> renders correctly 1`] = `
<main>
  <HeaderComponent />
  <h1>
    Home Page
  </h1>
</main>
`;
